artifacthub/library/general/requirednamespacequotas/1.0.0/template.yaml

```
@@ -0,0 +1,41 @@
+apiVersion: templates.gatekeeper.sh/v1
+kind: ConstraintTemplate
+metadata:
+  name: requirednamespacequota
+  annotations:
+    metadata.gatekeeper.sh/title: "Required Namespace Quota"
+    metadata.gatekeeper.sh/version: 1.0.0
+    description: >-
+      Ensures that all namespaces have a ResourceQuota defined.
+spec:
+  crd:
+    spec:
+      names:
+        kind: RequiredNamespaceQuota
```

```suggestion
        kind: K8sRequiredNamespaceQuota
```
Similar should be applied to other places like `name:` and for `package`.